ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -266,15 +268,27 @@ gcp_flush_finish([[maybe_unused]] WT_STORAGE_SOURCE *storage, [[maybe_unused]] W
 }
 
 static int
-gcp_file_exists(
-  WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, bool *file_exists)
+gcp_file_system_exists(
+  WT_FILE_SYSTEM *file_system, [[maybe_unused]] WT_SESSION *session, const char *name, bool *existp)
```

Comment:
This doesn't look right. Since you're definitely not using `session`, this is misleading. Not used anywhere else in WT either. I'd stick to `WT_UNUSED(session)`.

Reply:
I asked Sona to use the C++ convention for unused variables here.